src/register-tile-types.ts

```diff
@@ -1,5 +1,5 @@
 // import all tools so they are registered
-import "./models/tiles/unknown-content";
+import "./models/tiles/register-unknown-content";
```

Comment:
For consistency it should be called `unknown-content-registration`.   This also keeps all of the `unknown-*` files next to each other in tiles folder.